app/src/main/java/net/gsantner/opoc/frontend/GsSearchOrCustomTextDialog.java

```diff
@@ -84,9 +83,16 @@ public static class DialogOptions {
         @Nullable
         public GsCallback.a1<List<Integer>> positionCallback = null;
 
+        // For multi select
         public boolean isMultiSelectEnabled = false;
+        public Collection<Integer> preSelected = null;  // Indices of pre-selected items
+        public boolean showSelectAllButton = true;
+        public boolean showCountInOkButton = true;
+        public GsCallback.a1<Set<Integer>> selectionChangedCallback = null;
```

Comment:
Allows us to add a lot of dynamic functionality into the dialog. In this PR I have used it to implement Radio buttons